website/docs/reference/environments.md

```diff
@@ -45,11 +39,7 @@ Environments exist on a global level, so they are available to all projects with
 
 ## How to start using environments
 
-In order to start using environments you need to be on Unleash v4.2 or higher.
-
-If you are on v4.2, you also need to have the environment feature enabled (if you are using Unleash Hosted, please reach out on [contact@getunleash.io](mailto:contact@getunleash.io) if you want to start using environments.
-
-If you are on v4.3 or later, environments are already enabled for you.
+If you are on v4.2, you need to have the environment feature enabled (if you are using Unleash Hosted, please reach out on [contact@getunleash.io](mailto:contact@getunleash.io) if you want to start using environments. If you are on v4.3 or later, environments are already enabled for you.
```

Comment:
We can probably remove this paragraph entirely. There are no Unleash hosted customers on 4.2 (or anything below 6.2 .. 6.3?)